packages/webview_flutter/android/src/main/java/io/flutter/plugins/webviewflutter/FlutterWebViewClient.java

```diff
@@ -223,18 +226,19 @@ public void onPageFinished(WebView view, String url) {
 
       // This method is only called when the WebViewFeature.RECEIVE_WEB_RESOURCE_ERROR feature is
       // enabled. The deprecated method is called when a device doesn't support this.
+      @RequiresApi(api = Build.VERSION_CODES.LOLLIPOP)
```

Comment:
what happens if the method is called on a lower api version?

Reply:
The comment above should explain the expected behavior. If this method isn't supported, it should call the deprecated method below. This is according to: https://developer.android.com/reference/android/webkit/WebViewClient#onReceivedError(android.webkit.WebView,%20android.webkit.WebResourceRequest,%20android.webkit.WebResourceError)